lib/cli/options.js

```diff
@@ -25,6 +25,25 @@ const EXIT_CODES = require('../shared/exit-codes');
 const packageJSON = require('../../package.json');
 
 /**
+ * @summary The required number of CLI arguments
+ * @constant
+ * @private
+ * @type {Number}
+ */
+const REQUIRED_NUMBER_OF_ARGUMENTS = 1;
+
+/**
+ * @summary The first index that represents an actual option argument
+ * @constant
+ * @private
+ * @type {Number}
+ *
+ * @description
+ * The first arguments are usually the program executable itself, etc.
```

Comment:
> Maybe the name is confusing? My intention is that OPTIONS_INDEX_START equals the first index of argv that is a option to the program.

Yeah, I understood that, I just think the `@description` is confusing. Perhaps it'd simply be clearer if the `@description` here (and at https://github.com/resin-io/etcher/pull/1149/files#diff-1db67373c40ea3d2756fbc0768723000R54 ) was just removed. I don't think it adds anything.